### DISCUSSION TITLE
Smartmeter data availability

### DISCUSSION THREAD

**Discussion by nightfever77** [DarwinsBuddy/WienerNetzeSmartmeter#278] [General]
This is not related to the integration itself but to Wiener Netze: 
I am getting no data since 4 days. Even in the smart meter portal the consumption graph stops at 16.11.
Anyone else having that? 
I feel that Wiener Netze is really unreliable in providing smart meter data in general.

**Comment by reox** [comment]
This is sometimes the case and nothing you should worry about.
If you want to have a more reliable setup, you would need to activate the customer port on the smartmeter itself...

**Comment by nightfever77** [comment]
My smartmeter is in the underground garage. 
But I am planning to install a Shelly EM3 in my flat.

**Comment by DarwinsBuddy** [comment]
wiener netze sometimes fails on us. there have been reportedly outages of the api and the portal for even a couple of days. nothing we can do about it.
